server/routes/prisoner/handlers/overview.ts

```diff
@@ -1,14 +1,31 @@
 import { Request, Response } from 'express'
 import PrisonerService from '../../../services/prisonerService'
+import AdjustmentsService from '../../../services/adjustmentsService'
 
 export default class OverviewRoutes {
-  constructor(private readonly prisonerService: PrisonerService) {}
+  constructor(
+    private readonly prisonerService: PrisonerService,
+    private readonly adjustmentsService: AdjustmentsService,
+  ) {}
 
   GET = async (req: Request, res: Response): Promise<void> => {
     const { prisoner } = req
     const { token } = res.locals.user
-    const nextCourtEvent = await this.prisonerService.getNextCourtEvent(prisoner.bookingId as unknown as number, token)
+    const [nextCourtEvent, adjustments] = await Promise.all([
+      this.prisonerService.getNextCourtEvent(prisoner.bookingId as unknown as number, token),
+      this.adjustmentsService.getAdjustments(prisoner.prisonerNumber, token),
+    ])
 
-    res.render('pages/prisoner/overview', { prisoner, nextCourtEvent })
+    const aggregatedAdjustments = adjustments.reduce((previous: { [key: string]: number }, current) => {
+      const total = (previous[current.adjustmentTypeText] ?? 0) + current.daysTotal
+      let newAggregate = previous
+      if (total) {
+        // eslint-disable-next-line no-param-reassign
```

Comment:
not sure - but would it still work if we removed the if condition and would it resolve the eslint issue? e.g
`return ((previous[current.adjustmentTypeText] = total), previous)`